spawn dist/electron.exe ENOENT

win10 + 64
node.js 5.1.1 + 32
```json
  "dependencies": {
    "ffi": "^2.0.0"
  },
  "devDependencies": {
    "electron-prebuilt": "^0.36.0",
    "electron-rebuild": "^1.0.2"
  }
```
* exec
```bash
.\node_modules\.bin\electron-rebuild.cmd
```
* error info
```bash
spawn dist/electron.exe ENOENT
Error: spawn dist/electron.exe ENOENT
    at exports._errnoException (util.js:856:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:452:9)
    at process._tickCallback (node.js:366:17)
```

node_modules\electron-prebuilt\dist\electron.exe is exist

Tried this change and am getting:

Cannot find module 'npm/node_modules/node-gyp/bin/node-gyp'
Error: Cannot find module 'npm/node_modules/node-gyp/bin/node-gyp'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.require.resolve (module.js:389:19)
    at installNodeHeaders$ (/Users/steve/Desktop/electron/node_modules/electron-rebuild/lib/main.js:166:25)
    at tryCatch (/Users/steve/Desktop/electron/node_modules/babel-runtime/regenerator/runtime.js:72:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/steve/Desktop/electron/node_modules/babel-runtime/regenerator/runtime.js:334:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as throw] (/Users/steve/Desktop/electron/node_modules/babel-runtime/regenerator/runtime.js:105:21)
    at tryCatch (/Users/steve/Desktop/electron/node_modules/babel-runtime/regenerator/runtime.js:72:40)
    at invoke (/Users/steve/Desktop/electron/node_modules/babel-runtime/regenerator/runtime.js:146:20)
    at /Users/steve/Desktop/electron/node_modules/babel-runtime/regenerator/runtime.js:156:13
    at run (/Users/steve/Desktop/electron/node_modules/core-js/library/modules/es6.promise.js:104:47)

with node v5.1.1 and npm 3.3.12

--- a/src/cli.js
+++ b/src/cli.js
@@ -50,7 +50,7 @@ let nodeModuleVersion = null;
 if (!argv.n) {
   try {
     let pathDotText = path.join(argv.e, 'path.txt');
-    electronPath = fs.readFileSync(pathDotText, 'utf8');
+    electronPath = path.resolve(argv.e, fs.readFileSync(pathDotText, 'utf8'));
   } catch (e) {
     console.error("Couldn't find electron-prebuilt and no --node-module-version parameter set, always rebuilding");
   }

